### ISSUE TITLE
Restructure examples and create formal tests

### ISSUE THREAD

**Issue by Cole-Monnahan-NOAA** [noaa-afsc/afscOSA#8]
The package would benefit from some simpler examples that users could run more easily, and I think using simulated data would be a more straightforward approach. 

Some formal tests would also be welcome since the package is still in early development and we expect changes in the coming year.

**Comment by JaneSullivan-NOAA** [comment]
Love it. Could we add confidence intervals to the SDNRs as part of this PR?

**Comment by Cole-Monnahan-NOAA** [comment]
Yeah that's doable. I'll add a flag to turn them on/off too. Francis suggests only doing upper limit but I think for compositions we want a two-sided CI.

**Comment by Cole-Monnahan-NOAA** [comment]
Added to this PR https://github.com/noaa-afsc/afscOSA/pull/6
